wpiutil/src/main/native/include/wpi/uv/Handle.h

```diff
@@ -119,6 +119,29 @@ class Handle : public std::enable_shared_from_this<Handle> {
    */
   void Close() noexcept;
 
+  /**
+   * Set if the loop is closing.
+   *
+   * This is set during EventLoopRunner.Stop(), and can be used for other cases
+   * to indicate the loop should be closing. For instance for a uv_walk loop can
+   * use this to close existing handles.
+   *
+   * @param loopClosing true to set the loop currently in closing stages.
+   */
+  void SetLoopClosing(bool loopClosing) noexcept {
+    m_loopClosing = loopClosing;
+  }
+
+  /**
+   * Get the loop closing status.
+   *
+   * This can be used from closed() in order to tell if a closing loop is the
+   * reason for the close, or another reason.
+   *
+   * @return true if the loop is closing, otherwise false.
+   */
+  bool GetLoopClosing() const noexcept { return m_loopClosing; }
```

Comment:
IsLoopClosing?  Ala IsClosing?

Reply:
Ah yeah, I can rename it.

Reply:
I can't use IsClosing, because thats used for if this handle is closing. I can rename the function to IsLoopClosing though